test/integration/proposal-checks/IntegrationTestRoles.sol

```diff
@@ -54,6 +52,12 @@ contract IntegrationTestRoles is PostProposalCheck {
                     true,
                     string.concat("Expected ", addressNames[j], " to have role ", roles[i])
                 );
+                /// assert timelock forwards all calls on to core
+                assertEq(
+                    timelock.hasRole(role, addresses.mainnet(addressNames[j])),
+                    true,
+                    string.concat("Expected ", addressNames[j], " to have role ", roles[i], " in timelock")
+                );
```

Comment:
I think you should revert changes on this file, the unit tests of VoltTimelockController already check that access control is delegated to Core, it doesn't make much sense to do all these role checks on the timelock